src/cmd/internal/obj/loong64/obj.go

```diff
@@ -411,7 +411,7 @@ func preprocess(ctxt *obj.Link, cursym *obj.LSym, newprog obj.ProgAlloc) {
 				// The NOP is needed to give the jumps somewhere to land.
 				// It is a liblink NOP, not a hardware NOP: it encodes to 0 instruction bytes.
 				//
-				// We don't generate this for leafs because that means the wrapped
+				// We don't generate this for leaves because that means the wrapped
```

Comment:
Understood if you prefer to revert this. I prefer to use correct English at all times, however, some people might find `leafs` more suitable in code.

Reply:
Please use Gerrit for comments and code review, but not GitHub.

Reply:
https://go.dev/doc/contribute has contribution instructions.